src/shared/shortcut.js

```diff
@@ -108,11 +114,7 @@ export function installShortcut(
  * @param {(e: KeyboardEvent) => any} onPress - A function to call when the shortcut matches
  * @param {ShortcutOptions} [options]
  */
-export function useShortcut(
-  shortcut,
-  onPress,
-  { rootElement = document.body } = {}
-) {
+export function useShortcut(shortcut, onPress, { rootElement } = {}) {
```

Comment:
The default argument was removed here so that we only specify a default for `rootElement` in one place.